types/coin.go

```diff
@@ -662,7 +662,7 @@ func (coins Coins) IsEqual(coinsB Coins) bool {
 	coinsB = coinsB.Sort()
 
 	for i := 0; i < len(coins); i++ {
-		if !coins[i].IsEqual(coinsB[i]) {
+		if !coins[i].Equal(coinsB[i]) {
```

Comment:
Could you add a `Deprecated` comment in `IsEqual` for coin?
Maybe we should even make `IsEqual` call `Equal` instead, but that's a behavior change.

Can you as well add a changelog?